src/ontology/mondo-edit.obo

```diff
@@ -244236,9 +244236,15 @@ subset: ordo_disease {source="Orphanet:119"}
 subset: orphanet_rare {source="Orphanet:119"}
 subset: rare
 synonym: "autosomal recessive limb-girdle muscular dystrophy caused by mutation in SGCB" EXACT [MONDO:design_pattern]
-synonym: "Beta-sarcoglycan limb-girdle muscular dystrophy" RELATED [GARD:0003851]
+synonym: "autosomal recessive limb-girdle muscular dystrophy type 2E" EXACT [Orphanet:119]
+synonym: "beta-sarcoglycan limb-girdle muscular dystrophy" RELATED [GARD:0003851]
+synonym: "beta-sarcoglycan-related LGMD R4" EXACT [Orphanet:119]
+synonym: "beta-sarcoglycan-related limb-girdle muscular dystrophy R4" EXACT [Orphanet:119]
 synonym: "beta-sarcoglycanopathy" BROAD [DOID:0110279, Orphanet:119]
+synonym: "LGMD due to beta-sarcoglycan deficiency" EXACT [Orphanet:119]
+synonym: "LGMD type 2E" EXACT [Orphanet:119]
 synonym: "LGMD2E" EXACT ABBREVIATION [DOID:0110279, MONDO:Lexical, OMIM:604286, Orphanet:119]
+synonym: "LGMDR4" EXACT ABBREVIATION [OMIM:604286]
 synonym: "limb-girdle muscular dystrophy due to beta-sarcoglycan deficiency" EXACT [DOID:0110279, Orphanet:119]
 synonym: "limb-girdle muscular dystrophy type 2E" RELATED [GARD:0003851]
```

Comment:
This is also a synonym in Orphanet. Should `Orphanet:119` be added and this changed to `EXACT`?

Reply:
will do

Reply:
done